Zend/zend_execute_API.c

```diff
@@ -438,6 +438,12 @@ void shutdown_executor(void) /* {{{ */
 	zval *zv;
 #if ZEND_DEBUG
 	bool fast_shutdown = 0;
+#elif defined(__SANITIZE_ADDRESS__)
+	char *force_fast_shutdown = getenv("ZEND_FORCE_FAST_SHUTDOWN");
+	bool fast_shutdown = (
+		is_zend_mm()
+		|| (force_fast_shutdown && ZEND_ATOL(force_fast_shutdown))
+	) && !EG(full_tables_cleanup);
```

Comment:
I don't care about this a lot, but this doesn't fit well
- the introduced environment variable is going to work only for ASAN build
- it requires additional ``ASAN_OPTIONS=detect_leaks=0`` to be set
- in general, this is a hack for debugging...

May be changing the name to something including ASAN and/or DEBUG would make the meaning more explicit.
Anyway, I don't object.